stapi-fastapi/pyproject.toml

```
@@ -0,0 +1,36 @@
+[project]
+name = "stapi-fastapi"
+version = "0.6.0"
+description = "Sensor Tasking API (STAPI) with FastAPI"
+authors = [{ name = "Christian Wygoda", email = "christian.wygoda@wygoda.net" }]
```

@philvarner has done a lot of work here, let's add him to the authors.

```suggestion
authors = [
    { name = "Christian Wygoda", email = "christian.wygoda@wygoda.net" },
    { name = "Phil Varner", email = "phil@philvarner.com" }
]
```